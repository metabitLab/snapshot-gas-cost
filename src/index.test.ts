import snapshotGasCost from "./index";

describe('#snapshotGasCost', () => {
    it('works for a Promise<number>', async () => {
        await snapshotGasCost(Promise.resolve(100))
    })

    it('works for a number', async () => {
        await snapshotGasCost(100)
    })
})