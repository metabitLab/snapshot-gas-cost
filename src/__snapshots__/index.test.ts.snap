// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#snapshotGasCost works for a Promise<number> 1`] = `100`;

exports[`#snapshotGasCost works for a number 1`] = `100`;
